dev/ci/user-overlays/13377-LasseBlaauwbroek-fix-timeout.sh

```diff
@@ -0,0 +1,5 @@
+if [ "$CI_PULL_REQUEST" = "13377" ] || [ "$CI_BRANCH" = "fix-timeout" ]; then
```

Comment:
Since you use the overlay in 2 prs you need double the tests (as you can see  #13378 still has red ci)
Or you can wait for  #13377 to be merged